src/theme.ts

```diff
@@ -0,0 +1,257 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export const theme = {
```

Comment:
Why are these theme values not being pulled directly from `@zendeskgarden/react-theming`? See https://github.com/zendeskgarden/css-components/tree/master/packages/variables/src for comparison.

Reply:
`@zendeskgarden/react-theming` is a `devDependency` (well technically not listed as a dependency at all) for `css-variables`.

It's not installed by consumers as it's just `css`.

For this library it would need to be marked as a `dependency`. This would require consumers to include `styled-components`, `react`, `prop-types` and the other shared dependencies. That doesn't seem reasonable to consumers as this is strictly a Tailwind plugin and those could conflict with their own project.

Reply:
Further some values in `react-theming` aren't in the expected format for Tailwind to generate its style utilities.

Reply:
I've added some `react-theming` values here. Most use `DEFAULT_THEME` or `PALETTE` directly with some values still using static pixel values if they're base-1.